generators/client/templates/angular/webpack/_webpack.prod.js

```diff
@@ -23,6 +23,7 @@ const Visualizer = require('webpack-visualizer-plugin');
 const ngcWebpack = require('ngc-webpack');
```

Comment:
unused

Reply:
yes it is unused. we can remove it ( since ngc-webpack now uses ngtools/webpack behind the screens)